openy.info.yml

```diff
@@ -59,6 +59,7 @@ dependencies:
   - plugin
   - recaptcha
   - simple_menu_icons
+  - media_directories
```

Comment:
Looks like non needed on such high level. Let's keep dependency for openy_media only